utilities/pleiades/launch-dask.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+export LANG="en_US.utf8"
+export LANGUAGE="en_US.utf8"
+export LC_ALL="en_US.utf8"
+
+echo "Launching dask scheduler"
+s=`qsub launch-dask-scheduler.sh`
+sjob=${s}
+echo ${s}
+
+workers=${1:-4}
+echo "Launching dask workers (${workers})"
+for i in $(seq 1 ${workers}); do
+    qsub launch-dask-worker.sh
```

Comment:
You might want to look into array jobs for `qsub`.  I think you might want `qsub -t 1-$workers` here instead of the for loop.